### ISSUE TITLE
Patched Sur crashes when trying to access 'Advanced Configuration' in the 'Patch Kexts' window:

### ISSUE THREAD

**Issue by Blueysh** [Ben216k/Patched-Sur#3]
**Describe the bug**
Patched Sur crashes when I try to access the 'Advanced Configuration' window in 'Patch Kexts'. I was trying to do this to see if I could find a workaround to [this issue](https://github.com/Ben216k/Patched-Sur/issues/2), but I just got this instead.

**To Reproduce**
1. Go to 'Patch Kexts'
2. Click on 'Advanced Configuration'
3. If the error doesn't occur before doing this, give Root Permissions

**Expected behavior**
I should be able to access the Advanced Configuration window for patch kexts, wherever it is.

**Screenshots**
*Not uploading any, it's literally just a crash*

**Desktop (please complete the following information):**
 - Mac Model: 11" MacBook Air 2012
 - Version v1.0.1 (122)

**Additional context**
 The crash report gave me some app-specific info: `Patched_Sur/PatchKextsView.swift:527: Fatal error: Can't open advanced patches without a new version of the patcher.` If the latest version of the patcher can't support the advanced patches then why release the button before it can?

Here's a crash report that macOS provided:
[Pastebin](https://pastebin.com/UdUVLtVy)

**Comment by Blueysh** [comment]
@Ben216k Alright, so I've just booted back into macOS, extracted the Patched-Sur-Patches.zip file and dragged the `PatchSystem.sh` script into my Terminal. It told me that `NeededPatches.sh` was not found, so seeing as it had `/../NeededPatches.sh` in the path it tried to look in, I moved `NeededPatches.sh` to the previous folder. Then when I ran the `PatchSystem.sh` again, it said `/NeededPatches.sh: Permission denied`. What did I do wrong?
